cfgov/ask_cfpb/models/pages.py

```diff
@@ -166,15 +166,9 @@ def get_portal_cards(self):
             # If the portal page is live, link to it
             if portal_page.live:
                 url = portal_page.url
-            # Otherwise, link to the topic "see all" page if there is one
+            # Otherwise, skip this page
             else:
-                topic_page = topic.portal_search_pages.filter(
-                    language=self.language, live=True
-                ).first()
-                if topic_page:
-                    url = topic_page.url
-                else:
-                    continue  # pragma: no cover
+                continue  # pragma: no cover
```

Comment:
It looks like with these changes we're not doing anything if the portal page isn't live, right? If that's the case, should we just add `live=True` to the queryset filter around line 158 and get rid of this if/else?

Reply:
Nice catch! Updated in c5860a2b19